maintenance/pin-helper.sh

```diff
@@ -5,7 +5,22 @@ SUFFIX=${SUFFIX:-txt}
 pip install --upgrade pip
 pip install pip-compile-multi
 
-for dir in scriptworker_client iscript treescript ; do
+apt-get update
+apt-get install -y libsodium-dev
+
+for dir in \
+	addonscript \
+	balrogscript \
+	beetmoverscript \
+	bouncerscript \
+	iscript \
+	pushapkscript \
+	pushsnapscript \
+	scriptworker_client \
```

Comment:
At some point, we may want this pinned before the others, but I don't think that's necessary yet.